config/zones/BE.yaml

```diff
@@ -5,17 +5,54 @@ bounding_box:
   - - 6.15665815596
     - 51.4750237087
 capacity:
-  biomass: 1105
-  coal: 0
-  gas: 6915
-  geothermal: 0
-  hydro: 186
-  hydro storage: 1308
-  nuclear: 3928
-  oil: 455
-  solar: 8595
-  wind: 5315
-  battery storage: 50
+  biomass:
+    - datetime: '2023-01-01'
+      source: entsoe.eu
+      value: 1115.0
+    - datetime: '2024-01-01'
+      source: entsoe.eu
+      value: 1116.0
+  gas:
+    - datetime: '2023-01-01'
+      source: entsoe.eu
+      value: 6995.0
+    - datetime: '2024-01-01'
+      source: entsoe.eu
+      value: 6999.0
+  hydro:
+    datetime: '2023-01-01'
+    source: entsoe.eu
+    value: 186.0
+  hydro storage:
+    datetime: '2023-01-01'
+    source: entsoe.eu
+    value: 1308.0
```

Comment:
Last I heard they wanted to do it before new years but that didn't happen. But hopefully sooner rather than later but I don't have a timeline.

Reply:
Should be included in the next MoP according to what I last heard: https://www.entsoe.eu/data/transparency-platform/mop/

Reply:
TBH i don't know what to do about battery storage because i'm not sure where the value comes from. There are two things we can do:
- remove it (and assume it is aggregated in ENTSO-e unknown) and wait for ENTSOe to start reporting battery storage
- leave the value which adds significant manual work for maintenance as there more than one zone in that case

Reply:
For now i will go with the first option because the values are so small compared to the rest of the capacity breakdown it makes no difference

Reply:
Let's remove it. I think we should try and keep the capacity and production sources in line with each other as much as possible.

Reply:
yes i agree, and let's keep monitoring ENTSOE to see if they do add it :)